cli/script/command-parser.ts

```diff
@@ -306,7 +306,7 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
         isValidCommandCategory = true;
         isValidCommand = true;
         yargs.usage(USAGE_PREFIX + " link")
-            .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
+            .demand(/*count*/ 0, /*max*/ 0)  
```

Comment:
Same as for login command, I think max should be one because we have an undocumented option used for testing